### ISSUE TITLE
Demo1 build failure with Node.js v12.13.0

### ISSUE THREAD

**Issue by dilallkx** [InterDigitalInc/AdvantEDGE#32]
**Describe the bug**
The Demo1 example build fails when using the latest LTS version of Node.js (v12.13.0). It works with the previous LTS version v10.16.3.

**To Reproduce**
1. `cd ~/AdvantEDGE/examples/demo`
2. `./build-demo1.sh`
3. Observe the following error:

```
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/home/vagrant/AdvantEDGE/examples/demo1/src/demo-frontend/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (events.js:210:5)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:272:12)
gyp ERR! System Linux 4.4.0-166-generic
gyp ERR! command "/home/vagrant/.nvm/versions/node/v12.13.0/bin/node" "/home/vagrant/AdvantEDGE/examples/demo1/src/demo-frontend/node_modules/node-gyp/bin/node-gyp.js" "rebuild" "--verbose" "--libsass_ext=" "--libsass_cflags=" "--libsass_ldflags=" "--libsass_library="
gyp ERR! cwd /home/vagrant/AdvantEDGE/examples/demo1/src/demo-frontend/node_modules/node-sass
gyp ERR! node -v v12.13.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok 
Build failed with error code: 1
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! node-sass@4.11.0 postinstall: `node scripts/build.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the node-sass@4.11.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**Expected behavior**
Build should succeed.

**Screenshots**
N/A

**Scenario**
N/A

**AdvantEDGE versions**
```
$ meepctl version
Using repo config file: /home/englab/roymx/AdvantEDGE//.meepctl-repocfg.yaml
Using meepctl config file: /home/englab/.meepctl.yaml
{
  "name": "meepctl",
  "version": "1.2.0"
}
{
  "name": ".meepctl-repocfg.yaml",
  "version": "1.2.0"
}
```

```
$ node -v
v12.13.0

$ npm -v
6.12.0
```

**AdvantEDGE node**
```
$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 16.04.6 LTS
Release:        16.04
Codename:       xenial

$ uname -a
Linux W7280 4.15.0-65-generic #74~16.04.1-Ubuntu SMP Wed Sep 18 09:51:44 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
```

**Front-end node**
N/A

**Other nodes**
N/A

**Additional context**
N/A

**Comment by roymx** [comment]
Quick workaround: downgrade node.js to a supported version
ex: nvm install 10.16.3

**Comment by dilallkx** [comment]
Build successful with node version v12.19.0.
Issue fixed in AdvantEDGE release v1.6.0.
